src/App.jsx

```diff
@@ -186,7 +186,9 @@ export default function App() {
           />
           <Route path="payment-policy" element={<Payment_Policy />} />
           {/* Return and Cancellation page route */}
-          <Route path="epr" element={<EPRPage />} /> {/* EPR page route */}
+          <Route path="epr-compliance" element={<EPR_Page />} /> {/* EPR page route */}
```

Comment:
this is not working ... lets check it .. when I Try to go **_/epr-compliance_** then it is showing 404

https://vigy-bag.vercel.app/epr-compliance

try this endpoint locally and let me know